.gitignore

```diff
@@ -1,2 +1,3 @@
-/composer.lock
-/vendor/
```

Comment:
I don't agree. There are no vendor or composer.lock files below root in this project. There probably wont be in the future either.